Plugins/FastJet/include/Acts/Plugins/FastJet/Jets.hpp

```diff
@@ -83,6 +84,76 @@ class TrackJetBuilder {
   fastjet::ClusterSequence m_clusterSeq{};
 };
 
+class TruthJetBuilder {
+ public:
+  explicit TruthJetBuilder(const Acts::Vector4& fm) { m_fourMomentum = fm; }
+  const Acts::Vector4 getTruthJetFourMomentum() const { return m_fourMomentum; }
+
+ private:
+  Acts::Vector4 m_fourMomentum{0., 0., 0., 0.};
+};
+
+class JetProperties {
+ public:
+  /// Constructor; saves a reference to the jet
+  /// @param jet the jet
+  explicit JetProperties(const TruthJetBuilder& truthJet)
+      : m_truthJet{truthJet} {}
+
+  /// @brief Set the jet constituents
+  /// @param constituents the indices of the constituent tracks
+  void setConstituents(const std::vector<int>& constituents) {
+    m_constituents = constituents;
+  }
+  /// @brief Get the jet constituents
+  /// @return the indices of the constituent tracks
+  const std::vector<int>& getConstituents() const { return m_constituents; }
```

Comment:
If this class uses the truth jet builder, why the constitues are commented as "constituent tracks" ? 
They are particles, I would suppose.